### PR TITLE
Use a release tarball instead of a git tag

### PR DESCRIPTION
**Pull request by nightuser**
Fixes building if there are changes in dependencies.
It's the recommended way: see https://github.com/lxc/lxd/issues/6137 for details.

### CODE REVIEW

**Comment by brokenpip3** [comment]
Nice, ty, I'll test it

File: PKGBUILD [hunk 1]
```diff
@@ -23,42 +23,37 @@ optdepends=(
     'ceph: for ceph storage driver support'
 )
 options=('!strip' '!emptydirs')
```
Comment (brokenpip3) [inline PKGBUILD]: Add a blank line for better reading

File: PKGBUILD [hunk 2]
```diff
@@ -23,42 +23,37 @@ optdepends=(
...
 source=(
+    "https://${_lxd}/releases/download/${pkgname}-${pkgver}/${pkgname}-${pkgver}.tar.gz"
     "lxd.service"
     "lxd.socket"
     "dnsmasq-lxd.conf"
     "dnsmasq@lxd.service"
     "lxd.netctl"
     "dbus-dnsmasq-lxd.conf"
     "networkmanager-dnsmasq-lxd.conf"
+    "Remove-deps-update.patch"
```
Comment (brokenpip3) [inline PKGBUILD]: Please use lowercase

**Comment by brokenpip3** [comment]
Ty nightuser, this night i'll post on lxd forum for an approval of build method and then I'm going to push it to aur. Many thanks for your work, i tried the same way in the past days.

Ps add your contributor name in PKGBUILD ;)

**Comment by maikelwever** [comment]
First of all: thank you very much for taking the effort to make a pull request.

Looking over this, I don't see why the patch for the buildscript is needed. Seems like setting `export GOPATH="${srcdir}/${pkgname}-${pkgver}/_dist"` should ensure the Makefile does not attempt to git pull the dependencies, but this is only the case for lxd 3.16+.

**Comment by nightuser** [comment]
@maikelwever Yes, the patch is not needed. I somehow overlooked the first if-clause and thought that it either `git pull`s if there is already a repo or clones instead.

**Comment by brokenpip3** [comment]
@maikelwever lgfm, what do you think?
I'm going to squash and merge

**Comment by maikelwever** [comment]
Yeah looks good to me as well.

**Comment by brokenpip3** [comment]
Merged, well done @nightuser

**Pull request merged**